src/main/java/seedu/mtracker/console/AddInstrumentParser.java

```diff
@@ -61,12 +61,25 @@ public static boolean isValidPrice(String currentPrice) {
         try {
             Double.parseDouble(currentPrice);
         } catch (NumberFormatException e) {
-            ErrorMessage.displayAddInstrumentCurrentPriceError();
+            ErrorMessage.displayAddInstrumentPriceError();
             isValid = false;
         }
         return isValid;
     }
 
+    public static boolean isExpiryFilled(String expiryInput) {
+        boolean isFilled = true;
+        try {
+            if (expiryInput.isEmpty() | expiryInput.isBlank()) {
+                throw new IllegalArgumentException();
+            }

```

Comment:
I dont think the isBlank is needed cause the getUserCommand automatically trims whitespaces, so even if user puts in blank spaces the trimming will make it an empty string

Reply:
Ok so I will remove the 2nd .isBlank() condition then.